.drone.yml

```diff
@@ -91,7 +104,7 @@ pipeline:
     when:
       event: tag
       matrix:
-        CORE_VERSION: "10.0.0"
+        CORE_VERSION: "10.1.0"
```

Comment:
Changed release condition, is this OK? Should we use 10.2 instead? I think they are the same

Reply:
its ok like this